.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           name: Login into Docker hub
           command: docker login -u $DOCKER_USER -p $DOCKER_PASS
       - run: ./build.sh ubuntu
-      - run: ./build.sh ubuntu-systemd-xenial
+      - run: ./build.sh ubuntu-systemd-bionic
```

Comment:
Can we add both images?

Reply:
Why continue to keep the old one around, it it's never going to work without elevated privileges, and we're not going to use it in tests?

Reply:
Good point, derp.